src/main/java/internetshop/controllers/order/OrdersListController.java

```diff
@@ -14,7 +14,7 @@
 
 @WebServlet("/orders/all")
 public class OrdersListController extends HttpServlet {
```

Comment:
```suggestion
public class UserOrdersController extends HttpServlet {
```
краще використати такий неймінг, адже ми витягуємо не усі замовлення з бази а лише конкретного користувача